src/Core/Components/DataGrid/FluentDataGridRow.razor.cs

```diff
@@ -61,19 +61,22 @@ public partial class FluentDataGridRow<TGridItem> : FluentComponentBase, IHandle
 
     protected override void OnInitialized()
     {
+        RowId = $"r{Owner.GetNextRowId()}";
         Owner.Register(this);
     }
 
     public void Dispose() => Owner.Unregister(this);
 
     internal void Register(FluentDataGridCell<TGridItem> cell)
     {
+
+        cell.CellId = $"c{Owner.GetNextCellId()}";
         cells.Add(cell.CellId, cell);
     }
 
     internal void Unregister(FluentDataGridCell<TGridItem> cell)
     {
-        cells.Remove(cell.CellId);
+        cells.Remove(cell.CellId!);
```

Comment:
If VS says that this variable can be nullable. It's true in some cases 😊 Maybe a test to avoid the Remove in this case?

Reply:
I made it nullable during testing. I the end I reverted that so it isn't nullable anymore. Just forgot to remove the bang.